spec.json

```diff
@@ -1054,19 +1064,6 @@
                             ]
                         }
                     ]
-                },
-                {
-                    "propertyName": "Color",
-                    "values": [
-                        { "name" : "NoColor",  "value" : "0", "type" : "int", "description": [ "No color." ] },
-                        { "name" : "Black",    "value" : "1", "type" : "int", "description": [ "Black color." ] },
-                        { "name" : "Blue",     "value" : "2", "type" : "int", "description": [ "Blue color." ] },
-                        { "name" : "Green",    "value" : "3", "type" : "int", "description": [ "Green color." ] },
-                        { "name" : "Yellow",   "value" : "4", "type" : "int", "description": [ "Yellow color." ] },
-                        { "name" : "Red",      "value" : "5", "type" : "int", "description": [ "Red color." ] },
-                        { "name" : "White",    "value" : "6", "type" : "int", "description": [ "White color." ] },
-                        { "name" : "Brown",    "value" : "7", "type" : "int", "description": [ "Brown color." ] }
-                    ]
```

Comment:
Well, this PR is made against your own fork, so it can not be merged into the upstream repo. You'll need to make a new PR against upstream. I could also pull your changes myself if you think it would be easier for you?